docs/can-canjs/canjs.md

```diff
@@ -3,15 +3,17 @@
 @outline 0
 @package ../../package.json
 @templateRender <% %>
-@description CanJS is an evolving and improving set of client side JavaScript architectural libraries that balances innovation and stability. It targets experienced developers building complex applications with long futures ahead of them.   
-
-<img
-    srcset="docs/images/home/Home-Tortoise-bw.png 1x, docs/images/home/Home-Tortoise-bw-x2.png 2x"
-    src="docs/images/home/Home-Tortoise-bw.png"
-    style="float:right; padding-right: 40px;"/>
-<img
-    srcset="docs/images/home/Home-Hare-bw.png 1x, docs/images/home/Home-Hare-bw-x2.png 2x"
-    src="docs/images/home/Home-Tortoise-bw.png" style="padding-left: 40px;"/>
+@description CanJS is an evolving and improving set of client side JavaScript architectural libraries that balances innovation and stability. It targets experienced developers building complex applications with long futures ahead of them.
+
+<div class="hero-images">
+    <img
+        srcset="docs/images/home/Home-Tortoise-bw.png 1x, docs/images/home/Home-Tortoise-bw-x2.png 2x"
+        src="docs/images/home/Home-Tortoise-bw.png"
+        style="float:right; padding-right: 40px;"/>
+    <img
+        srcset="docs/images/home/Home-Hare-bw.png 1x, docs/images/home/Home-Hare-bw-x2.png 2x"
+        src="docs/images/home/Home-Tortoise-bw.png" style="padding-left: 40px;"/>
```

Comment:
Just FYI you didn’t remove the inline styles. If you added them to the theme, then they’ll be redundant, otherwise we can merge this as-is.